WordPress.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -24,26 +24,26 @@
         "repositoryURL": "https://github.com/mattgallagher/CwlCatchException.git",
         "state": {
           "branch": null,
-          "revision": "35f9e770f54ce62dd8526470f14c6e137cef3eea",
-          "version": "2.1.1"
+          "revision": "3b123999de19bf04905bc1dfdb76f817b0f2cc00",
```

Comment:
Could we remove these package version bumps from this PR? Thanks!

Reply:
For sure! I have removed the package version bumps